crypto/dkg_jointfeldman.go

```diff
@@ -202,10 +199,7 @@ func (s *JointFeldmanState) HandleBroadcastMsg(orig int, msg []byte) error {
 	for i := index(0); int(i) < s.size; i++ {
 		err := s.fvss[i].HandleBroadcastMsg(orig, msg)
 		if err != nil {
-			if IsInvalidInputsError(err) {
-				newInvalidInputsError("handle message has failed: %s", err)
-			}
```

Comment:
`HandleBroadcastMsg` can return a `invalidInputError` error or another error. If it is `invalidInputError`, it would be wrapped by the `fmt.Errorf` error and `isInvalidInputError` returns `true`.